app/src/main/java/com/techcourse/support/web/adapter/HandlerAdapters.java

```diff
@@ -0,0 +1,29 @@
+package com.techcourse.support.web.adapter;
+
+import com.techcourse.support.web.resolver.ViewResolvers;
+import java.util.ArrayList;
+import java.util.List;
+import webmvc.org.springframework.web.servlet.mvc.HandlerAdapter;
+
+public class HandlerAdapters {
+
+    private final List<HandlerAdapter> adapters = new ArrayList<>();
+
+    public void initialize(final ViewResolvers viewResolvers) {
+        final ManualHandlerMappingAdapter manualHandlerMappingAdapter = new ManualHandlerMappingAdapter(viewResolvers);
+        final AnnotationHandlerAdapter annotationHandlerAdapter = new AnnotationHandlerAdapter();
```

Comment:
사소한 부분이긴 한데, 변수 선언을 할 때 인터페이스를 통해 선언하는 것은 어떤가요?
만들어진 두 객체가 HandlerAdapter의 구현체라는게 더 잘 보일 것 같아요.

```suggestion
        final HandlerAdapter manualHandlerMappingAdapter = new ManualHandlerMappingAdapter(viewResolvers);
        final HandlerAdapter annotationHandlerAdapter = new AnnotationHandlerAdapter();
```

Reply:
맞습니다 이거는 제가 놓친 부분이네요 리펙토링 하도록 하겠습니다 감사합니다